x-pack/plugins/apm/public/components/app/Settings/custom_link/custom_link_table.tsx

```diff
@@ -119,9 +119,9 @@ export function CustomLinkTable({ items = [], onCustomLinkSelected }: Props) {
         }
         items={filteredItems}
         columns={columns}
-        initialPageSize={10}
         initialSortField="@timestamp"
         initialSortDirection="desc"
+        hidePerPageOptions={false}
```

Comment:
It could be just me but I don't really like to set a property to `false` in order to **show** it. Not really a big deal but I'd do:

```suggestion
        showPerPageOptions={true | false}
```